usd/src/sdf_parser/Geometry.cc

```diff
@@ -380,6 +383,43 @@ namespace usd
         pxr::GfVec3f(meshMax.X(), meshMax.Y(), meshMax.Z()));
       usdMesh.CreateExtentAttr().Set(extentBounds);
 
+      int materialIndex = subMesh->MaterialIndex();
+      if (materialIndex != -1)
+      {
+        const auto material = ignMesh->MaterialByIndex(materialIndex);
+        const sdf::Material materialSdf = sdf::usd::convert(material);
+        std::string materialPath;
+        UsdErrors materialErrors = ParseSdfMaterial(
+          &materialSdf, _stage, materialPath);
+        if (!materialErrors.empty())
+        {
+          errors.push_back(UsdError(
+            sdf::usd::UsdErrorCode::SDF_TO_USD_PARSING_ERROR,
+            "Unable to parsematerial"));
```

Comment:
nit: missing a space between words, capitalization optional

"Unable to parse Material" / "Unable to parse material"